lib/cfer/core/stack.rb

```diff
@@ -159,6 +159,17 @@ def to_cfn
       to_h.to_json
     end
 
+    # Includes template code from one or more files, and evals it in the context of this stack.
+    # Filenames are relative to the file containing the invocation of this method.
+    def include_template(*files)
+      calling_file = caller.first.split(/:\d/,2).first
```

Comment:
@seanedwards Haha, yeah, luckily ruby 1.9 onward implements `require_relative` via a hack in C instead (http://apidock.com/ruby/Kernel/require_relative). This ruby implementation was a backport to 1.8.